src/plugins/ai_assistant_management/observability/public/routes/components/search_connector_tab.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiLink, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useAppContext } from '../../hooks/use_app_context';
+
+export const SELECTED_CONNECTOR_LOCAL_STORAGE_KEY =
+  'xpack.observabilityAiAssistant.lastUsedConnector';
+
+export function SearchConnectorTab() {
+  const { application } = useAppContext();
+  const url = application.getUrlForApp('enterprise_search', { path: '/content/connectors' });
+
+  return (
+    <>
+      <EuiText>
+        {i18n.translate(
+          'aiAssistantManagementObservability.searchConnectorTab.searchConnectorsEnablesYouTextLabel',
+          {
+            defaultMessage:
+              'Search connectors enables you to index content from external sources thereby making it available for the AI Assistant. This can greatly improve the relevance of the AI Assistant’s responses.',
```

Comment:
```suggestion
              'Search connectors enable you to index content from external sources and make it available to the AI Assistant. This can greatly improve the relevance of the AI Assistant’s responses.',
```

nit copyedit

Reply:
> As a side-note, 'Search connectors' seems like a very awkward term. We just call them connectors. If you need to differentiate them from alerting connectors, maybe content connectors works better?

We call them "Elastic connectors" in the docs, not sure what term want to use either, but maybe linking to the [docs overview page](https://www.elastic.co/guide/en/enterprise-search/current/connectors.html) would help? We also seem to be using _Elasticsearch connectors_ here.

Options:

- Elastic connectors
- Elasticsearch connectors
- Third-party connectors

Reply:
I prefer to stay consistent with what's already there. Sounds like "Elastic connectors" might be the best then?

Reply:
Thanks for the copy change!

Reply:
Cool, no problem!